static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -23,13 +24,16 @@ class ReleasesConfirm extends Component {
   }
 
   onRevertClick() {
+    this.props.triggerGAEvent("click-revert");
     this.props.cancelPendingReleases();
     this.setState({
       showDetails: false
     });
   }
 
   onApplyClick() {
+    this.props.triggerGAEvent("click-save");
```

Comment:
Same with this one

Reply:
Will keep this where it is, to avoid triggering a "fake" `click-revert`